components/MapComponentView.js

```diff
@@ -85,26 +91,47 @@ export default class MapComponentView extends React.Component {
       : MAP_TYPES.NONE;
     }
 
+    //TODO MODULARIZE IN locationClient.js
+    getLocationApiCall(beacons) {
+      return fetch(GET_LOCATION_SERVICE_URL, {
+        method: 'POST',
+        headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          device_id: getUniqueId(),
+          nearby_stands: getNearbyStands(beacons)
+        }),
+      })
+      .then((response) => response.json())
+      .then((responseJson) => {
+        this.setState({
+          locationMarker:[responseJson]
+        });
+        this.fitAllMarkers();
+      })
+      .catch((error) => {
+        console.error(error);
```

Comment:
Evitemos romper en dev
```suggestion
        console.log(error);
```